Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -2329,27 +2329,31 @@ public static partial class PythonOps {
         /// If type is a type then value can either be an instance of type,
         /// a Tuple, or a single value.  This case is handled by EC.CreateThrowable.
         /// </summary>
-        public static Exception MakeException(CodeContext/*!*/ context, object type, object value, object traceback) {
-            Exception e = MakeExceptionWorker(context, type, value, traceback, false);
+        public static Exception MakeException(CodeContext/*!*/ context, object type, object value, object traceback, object cause) {
+            Exception e = MakeExceptionWorker(context, type, value, traceback, cause, false);
             e.RemoveFrameList();
             return e;
         }
 
-        private static Exception MakeExceptionWorker(CodeContext/*!*/ context, object type, object value, object traceback, bool forRethrow) {
+        private static Exception MakeExceptionWorker(CodeContext/*!*/ context, object type, object value, object traceback, object cause, bool forRethrow) {
             Exception throwable;
             PythonType pt;
 
             if (type is PythonExceptions.BaseException) {
+                // Set nested python exception
+                var _cause = cause as PythonExceptions.BaseException;
+                ((PythonExceptions.BaseException)type).SetCause(_cause);
+
```

Comment:
Does anyone has an Idea, who to access an *implicit exception here? For example for:

```
try:
    int("abc")
except Exception as ex:
    raise NameError
```

If `raise NameError` is executed, I need the instance if the exception which is thrown above. When using the `from` keyword I can pass the exception using the new implementation of cause, but not without `from`...